servers/src/main/java/tachyon/master/file/meta/Inode.java

```diff
@@ -30,7 +30,7 @@
     private long mLastModificationTimeMs;
     private String mName;
     private long mParentId;
-    private boolean mPersisted;
+    private FilePersistenceState mPersistenceState;
```

Comment:
It seems odd to use the type "FilePersistenceState" in the Inode (which is inherited from by InodeDirectory). Why not just PersistenceState?